test/Npgsql.Specification.Tests/NpgsqlDbFactoryFixture.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Data.Common;
+using AdoNet.Specification.Tests;
+
+namespace Npgsql.Specification.Tests
+{
+    public class NpgsqlDbFactoryFixture : IDbFactoryFixture
+    {
+        public DbProviderFactory Factory => NpgsqlFactory.Instance;
+
+        const string DefaultConnectionString =
+            "Server=localhost;User ID=npgsql_tests;Password=npgsql_tests;Database=npgsql_tests;Timeout=0;Command Timeout=0";
+
+        public string ConnectionString =>
+            Environment.GetEnvironmentVariable("NPGSQL_TEST_DB") ?? DefaultConnectionString;
+
+        //public override string CreateHexLiteral(byte[] value) => $@"E'{string.Join("", value.Select(x => @"\x" + x.ToString("X2")))}'::bytea";
```

Comment:
Delete commented-out code?

Reply:
Yes, done. At some point I'll look at moving more ADO-generic tests from Npgsql's own test suite, that will make clear what general infrastructure requirements we have for the fixture etc.